proposals/2161/registry.md

```diff
@@ -111,7 +111,7 @@ airflow/
 	{
 		producer: {
 			root_doc_URL: “https://airflow/doc”
-			produced_facets: [
+			facets: [
```

Comment:
Thank you for pointing out the inconsistency. I agree we should update the doc so it is consistently using consumed_facets and produced_facets.

I think it is worth distinguishing consumed_facets and produced_facets as they have very different meanings.
- produced_facets: the producer is often defining the facet and making it available for others to consume. From a testing perspective, we'll verify those facets are produced and are correct against the schema.
- consumed_facets: this implies a dependency. Consuming the facet that someone else is producing. From a testing perspective, we'll provide example facets for consumer to test against.